.gitignore

```diff
@@ -11,6 +11,7 @@ Thumbs.db
 node_modules/
 sftp-config.json
 .push.settings.json
+.push.settings.jsonc
```

Comment:
files like that should be added to global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore. Then You don't need to add them to every project gitignore file